### ISSUE TITLE
Update R-Wrapping from SimpleITK

### ISSUE THREAD

**Issue by zarquon42b** [kaspermarstal/SimpleElastix#33]
Hi, in the upstream version of SimpleITK R wrapping has become quite smooth lately. Would it be hard to merge these changes into SimpleElastix?

Best 
Stefan

**Comment by kaspermarstal** [comment]
Hi Stefan

Merging these changes is indeed planned. I am waiting on the next release of SimpleITK before incorporating the changes because SimpleElastix is based on SimpleITK's release branch for stability. You can speed up the process by providing feedback on the SimpleITK RC when it comes out :blush: 

Kasper

**Comment by zarquon42b** [comment]
Awesome, thanks.
Stefan

**Comment by kaspermarstal** [comment]
New R bindings have now been merged into SimpleElastix.
